core/domain/exp_jobs_one_off_test.py

```diff
@@ -397,6 +418,29 @@ def test_nonhuman_committers_not_counted(self):
             feconf.MIGRATION_BOT_USERNAME,
             exploration_summary.contributor_ids)
 
+    def test_no_action_is_performed_for_deleted_exploration(self):
+        """Test that no action is performed on deleted explorations."""
+
+        self.signup(self.EMAIL_A, self.USERNAME_A)
+        user_a_id = self.get_user_id_from_email(self.EMAIL_A)
+
+        exp_id = '100'
+        exploration = self.save_new_valid_exploration(
+            exp_id, user_a_id)
+        exp_services.delete_exploration(user_a_id, exp_id)
+
+        # Run the ExpSummariesContributorsOneOff job.
+        job_id = (
+            exp_jobs_one_off.ExpSummariesContributorsOneOffJob.create_new())
+        exp_jobs_one_off.ExpSummariesContributorsOneOffJob.enqueue(job_id)
+        self.process_and_flush_pending_tasks()
+
+        with self.assertRaisesRegexp(
+            base_models.BaseModel.EntityNotFoundError,
+            'Entity for class ExpSummaryModel with id 100 not found'
+        ):
```

Comment:
This can be placed on the previous line right?

Reply:
Done.